gemfiles/rubocop.gemfile.lock

```diff
@@ -0,0 +1,44 @@
+GEM
```

Comment:
Oh, this file shouldn't be here. Looks like we're missing the `gemfiles/*.lock` entry in the `.gitignore`. Could you please add one?